orchestrators/orchestrator.go

```diff
@@ -20,9 +22,7 @@ func GetOrchestrator(c *handler.Conplicity) Orchestrator {
 
 	switch orch {
 	case "docker":
-		return &DockerOrchestrator{
-			Handler: c,
-		}
+		return InitDocker(c)
```

Comment:
Or maybe `NewDockerOrchestrator()` so we can use more standard naming for each orchestrator.